src/Makefile.am

```diff
@@ -2178,9 +2178,9 @@ tests_testHttpReply_SOURCES = \
 	tests/stub_cache_cf.cc \
 	cache_cf.h \
 	tests/stub_cache_manager.cc \
-	tests/stub_client_side.cc \
 	cbdata.cc \
 	cbdata.h \
+	tests/stub_client_side.cc \
```

Comment:
This was picked by a source-format. I can revert it, but it will be added back by the source-format.sh . Are you sure?

Reply:
FWIW, my general recommendation is to revert unrelated formatting changes whenever possible, as soon as possible (ideally before git commit or at least before git push). It does not matter that those changes will be added later in a separate future formatting PR. What does matter is that they are unrelated and, hence, increase current PR complexity and porting overheads (sometimes slightly, sometimes by a lot).

Eventually, there will be no such unrelated changes because we will enforce formatting on every PR, but we need better GitHub integration for that to happen.

Reply:
(unrelated) The changes to source-maintenance.sh in PR 830 (and some followup ideas I have) might make it fast enough to enforce formatting as a pre-commit hook, if someone has the right astyle installed

Reply:
Reverted in commit 22e8f065a (will be pushed shortly)

Reply:
> (unrelated) The changes to source-maintenance.sh in PR 830 (and some followup ideas I have) might make it fast enough to enforce formatting as a pre-commit hook, if someone has the right astyle installed

Sounds good. And Semaphore CI already has the code to check formatting, but it is currently essentially disabled, primarily because the Project policy makes such formatting optional. All that is easy though... The GitHub integration I was [talking](https://github.com/squid-cache/squid/pull/667#discussion_r642532273) about is the complex missing part -- it would allow the author to tell the bot to properly format the PR (without requiring the developer to install a lot of software completely unrelated to their work). This is not rocket science either, of course, but crossing that barrier properly requires some serious development work (which will allow a lot of other handy features). Until that missing piece is in, we should not _require_ PR formatting IMO.